atest/testdata/standard_libraries/builtin/log.robot

```diff
@@ -163,3 +163,10 @@ Log To Console
     Log To Console    stderr äö w/ newline    stdERR
     Log To Console    ...line äö   stdout    continue without newlines
     Log To Console    ${42}
+    Log to console    test middle align with star padding    format=*^60
+    Log to console    test right align with hash padding    format=#>32
+    Log To Console    test-with-spacepad-and-weird-characters+%?,_\>~}./asdf    format=>30
+    Log To Console    message starts here,    format=>44  no_newline=true
+    Log To Console    this sentence should be on the same sentence as "message starts here"
+    Log to console    test log to stderr    format=>44  stream=stdERR
+    
```

Comment:
Perhaps a separate test like "Log To Console with formatting" should be added with these changes. There could be such a test also on the `atest/robot` side.

There's also some strange whitespace at the end that should be removed.